site/src/components/components/ComponentsList.tsx

```diff
@@ -56,11 +56,16 @@ const componentsListSorting = (
   );
 };
 
+const isExternalLink = (url: string) => /(http(s?)):\/\//i.test(url);
+
 const ComponentNameData = ({ component }: { component: ComponentDetails }) => {
-  const { devStatus, name, storybookUrl } = component;
+  const { name, docsUrl } = component;
 
-  return devStatus === ComponentStatus.READY && storybookUrl ? (
-    <Link href={storybookUrl}>
+  return docsUrl ? (
+    <Link
+      href={docsUrl}
+      target={isExternalLink(docsUrl) ? "_blank" : undefined}
```

Comment:
We don't need `_target="blank"` here - linking to another website is no reason to force opening a new tab. Our own Link component documentation also advises against doing this.

Reply:
We wanted to differentiate the storybook links from the internal ones by using the tear out icon (which only makes sense if we open on a new page) so maybe we need a different approach? I can add the storybook icon instead cc @origami-z

Reply:
Can be any visual difference, so user wouldn't be surprised to not finding them on the side bar.

Reply:
Yeah, a Storybook icon could work. Alternatively, just append " (on Storybook)" or something similar to the link text. E.g. "AG Grid theme (on Storybook)".

It's unambiguous and potentially more meaningful for screenreader users, who might be listening to the link texts out of context.

Reply:
I've added the icon which I think looks a bit better on the table but let me know your thoughts 😄